aiida/cmdline/commands/cmd_config.py

```diff
@@ -66,7 +66,10 @@ def verdi_config_list(ctx, prefix, description: bool):
     config: Config = ctx.obj.config
     profile: Profile = ctx.obj.profile
 
-    option_values = config.get_options(profile.name)
+    if not profile:
+        echo.echo_warning('no profiles configured: run `verdi setup` to create one')
```

Comment:
is that so?
doesn't this mean that there is no default profile configured?

Reply:
can you have profiles configured without a default profile set?

Reply:
`verdi setup` automatically adds the created profile as the default and there is at least no "user based" method to unset the default (you can just set a different profile) so I think this is accurate. Happy for you to suggest an alternative message though?

Reply:
hm... I vaguely remember having this situation some time ago but you're right- in the end the vast majority of the cases will be because there are no profiles at all. Let's keep it.

Reply:
@chrisjsewell What happens if you delete the default profile?

Reply:
yeh I guess thats an edge case. Lokking at `Config.remove_profile`, it doesn't appear to reset `default_profile`, if the profile being removed is the default.
Perhaps it should do that?
abd perhaps set another profile as the default if there is still existing profiles

Reply:
Unless someone wants to enforce an explicit specification of the profile, e.g. to avoid accidentally modifying one of the profiles.
In the end it may still the warning message here that is not quite accurate

Reply:
I'll leave it up to you if you want to open a PR to modify it 😉